src/test/mir-opt/const_allocation3/rustc.main.ConstProp.after.mir

```diff
@@ -26,18 +26,18 @@ fn  main() -> () {
     }
 }
 
-alloc2 (const_allocation3[317d]::FOO[0]) size: 8, align: 8 {
-    ├──────alloc8+0───────┤ 
+alloc2 (static: const_allocation3[317d]::FOO[0], size: 8, align: 8) {
```

Comment:
Could still use `tcx.def_path_str` instead of `{:?}`-printing `DefId`.

Reply:
I tried `def_path_str`, but that omits the crate name, so when a static comes from another crate, won't we confuse it with a local static?

Reply:
> so when a static comes from another crate, won't we confuse it with a local static?

No, it would then have the crate name at the start of the path. It's the same printing as all of `ty::print::pretty` that you rely on for types and constants already.